qiskit/qobj/qobj.py

```diff
@@ -28,16 +29,39 @@
 
 class QobjSchema(BaseSchema):
     """Schema for Qobj."""
-
     # Required properties.
     qobj_id = String(required=True)
+    schema_version = String(required=True, missing=QOBJ_VERSION)
+
+    # Required properties depend on Qobj type.
     config = Nested(QobjConfigSchema, required=True)
     experiments = Nested(QobjExperimentSchema, required=True, many=True)
     header = Nested(QobjHeaderSchema, required=True)
-    type = String(required=True,
-                  validate=OneOf(QobjType.QASM.value,
-                                 QobjType.PULSE.value))
-    schema_version = String(required=True, missing=QOBJ_VERSION)
+    type = String(required=True, validate=OneOf(choices=(QobjType.QASM, QobjType.PULSE)))
+
+
+class QasmQobjSchema(QobjSchema):
+    """Schema for QasmQobj."""
+
+    # Required properties.
+    config = Nested(QasmQobjConfigSchema, required=True)
+    experiments = Nested(QasmQobjExperimentSchema, required=True, many=True)
+    header = Nested(QobjHeaderSchema, required=True)
```

Comment:
Can you remove this attribute, as it is already present in the parent `QobjSchema` with the same definition, and also remove it from `PulseQobjSchema`?